tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_changes.py

```diff
@@ -26,14 +26,23 @@
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 configurations_path = os.path.join(test_data_path, 'wazuh_conf.yaml')
 testdir1, testdir2 = test_directories
+subdir1 = os.path.join(PREFIX, 'testdir3/subdir1')
+subdir2 = os.path.join(PREFIX, 'testdir3/subdir2')
```

Comment:
`testdir3` should be included in `test_directories` so it can be automatically removed when the test finishes.